lib/rules/no-multi-spaces.js

```diff
@@ -70,7 +70,7 @@ module.exports = function(context) {
 
             var source = context.getSource(),
                 allComments = context.getAllComments(),
-                pattern = /[^\n\r\u2028\u2029 ] {2,}/g,  // note: repeating space
+                pattern = /[^\s].? {2,}/g,  // note: repeating space
```

Comment:
I've updated it to ignore the list of whitespace and tabs before the first character. Since JS regex doesn't support negative lookbehinds, the other alternative would be to trim leading tabs and then do the pattern matching, but that approach sounds superfluous to me.